tests/component/_task_modules/test_triggers.py

```diff
@@ -24,6 +25,21 @@ def ci_count_edges_task(task, sim_9185_device) -> nidaqmx.Task:
     return task
 
 
+@pytest.fixture()
+def sim_6363_ai_voltage_task(task, sim_6363_device):
+    """Gets AI voltage task."""
+    task.ai_channels.add_ai_voltage_chan(sim_6363_device.ai_physical_chans[0].name)
+    yield task
```

Comment:
Use `yield` when the fixture needs to do cleanup. In this case, the `task` fixture already does the cleanup, so use `return task`.

A yield fixtures would have a return type hint of `-> Generator[Task, None, None]`. For return, do `-> Task`.